2996_reflection/reflection.md

```diff
@@ -5399,7 +5427,7 @@ consteval bool is_data_member_spec(info r);
 for every `@$r$~$k$~@` in `mdescrs`.
 
 [#]{.pnum} *Effects*:
-Defines `class_type` with properties as follows:
+Produces an injected definition ([expr.const]) of `class_type`, whose locus is immediately after the manifestly constant-evaluated expression whose evaluation is producing the definition, with properties as follows:
```

Comment:
Should this be injected declaration?

Reply:
Since a definition is a declaration, I figured it's probably sufficiently clear to abbreviate "injected declaration that is a definition" as "injected definition" (since this is all so verbose already). What do you think?

Reply:
I prefer things to be a bit more searchable ... _but_, CWG might disagree with me entirely; so it's probably not worth splitting hairs over.

Reply:
Good point; done.